config_full.json.example

```diff
@@ -62,8 +62,8 @@
             "refresh_period": 1800
         },
         {"method": "PrecisionFilter"},
-        {"method": "PriceFilter", "low_price_ratio": 0.01
-        }
+        {"method": "PriceFilter", "low_price_ratio": 0.01},
+        {"method": "SpreadFilter", "max_spread_ratio": 0.0025}
```

Comment:
would we want the sample in the configuration to be aligned to the documentation (so it's 0.005)?